samples/esb/esb_prx/src/main.c

```diff
@@ -131,10 +131,8 @@ int clocks_start(void)
 		}
 	} while (err == -EAGAIN);
 
-#if defined(NRF54L15_XXAA)
-	/* MLTPAN-20 */
-	nrf_clock_task_trigger(NRF_CLOCK, NRF_CLOCK_TASK_PLLSTART);
-#endif /* defined(NRF54L15_XXAA) */
+	nrf_lrcconf_clock_always_run_force_set(NRF_LRCCONF000, 0, true);
+	nrf_lrcconf_task_trigger(NRF_LRCCONF000, NRF_LRCCONF_TASK_CLKSTART_0);
```

Comment:
Does this workaround apply as well to nRF54L15 and replace the workaround for MLTPAN-20?

Reply:
No, this workaround applies to nRF54H20.
MLTPAN-20 at this place was redundant, for the `CONFIG_CLOCK_CONTROL_NRF2` configuration it was never included.
MLTPAN-20 remained for the `CONFIG_CLOCK_CONTROL_NRF` configuration.